extensions/amp-subscriptions/0.1/test/test-dialog-renderer.js

```diff
@@ -29,12 +29,15 @@ describes.realWin('amp-subscriptions DialogRenderer', {
   let dialogMock;
   let renderer;
   let dialog0, dialog1, dialog2, dialog3;
+  let actionRendererStub;
 
   beforeEach(() => {
     win = env.win;
     doc = win.document;
     ampdoc = env.ampdoc;
     templatesMock = sandbox.mock(Services.templatesFor(win));
+    actionRendererStub = sandbox.stub(LocalRenderer, 'renderActions').callsFake(
```

Comment:
Not sure if this will actually work as a testing in all cases. The module stubbing is not possible. I think you need to re-factor it as a separate API on the instance of Renderer class and pass that instance to the dialog renderer. Then you can easily stub it here.